src/functions-and-arrays.js

```diff
@@ -28,9 +92,34 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
+function uniquifyArray(arr) {
+  if (arr.length === 0) {
+    return null;
+  }
+  let uniqueArray = [];
+  arr.forEach(function (element) {
+    if (!uniqueArray.includes(element)) {
+      uniqueArray.push(element)
+    }
+  });
+  return uniqueArray;
+}
+
+uniquifyArray(wordsUnique);
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
+function doesWordExist(arr, word) {
+  if (arr.length === 0) {
+    return null;
+  } else if (arr.length === 1 && arr.includes(word)) {
+    return true;
+  } else if (arr.includes(word)) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Another possible solution would be:
`
for (let occurrence of arrayOfWords) {
    if (occurrence === word) return true;
  }
  return false;
`